autoload/fzf/vim.vim

```diff
@@ -41,7 +41,8 @@ if s:is_win
   else
     let s:bin.preview = fnamemodify(s:bin.preview, ':8')
   endif
-  let s:bin.preview = 'bash '.escape(s:bin.preview, '\')
+    let git_bash_bin = 'bash /mnt/'.substitute(s:bin.preview, '\([A-Z]\):', '\L\1\e', '')
```

Comment:
Why is `/mnt/` here? What if git with another method? What if the bash executable is not git bash?

Reply:
The best fix would probably be to write a Powershell script that would be use for windows.
This would require to bypass the execution policy, [see](https://riptutorial.com/powershell/example/20107/bypassing-execution-policy-for-a-single-script).

If you are ok with this, i can create another PR with a Powershell script.

Reply:
I'm ok with it but the powershell script should be cross-platform if possible so that it can be tested on Linux.